content/en/api/downtimes/downtimes_schedule.md

```diff
@@ -26,6 +26,9 @@ external_redirect: /api/#schedule-monitor-downtime
 * **`recurrence`** [*optional*, *default*=**None**]:
     An object defining the recurrence of the downtime with a variety of parameters:
     *   **`type`** the type of recurrence. Choose from: `days`, `weeks`, `months`, `years`.
+      *   **`rrule`** standard for defining recurring events. For example to have a recurring event each month, select a type of `rrule` and a `FREQ` of `MONTHLY`. Any RRULE according to RFC is supported from this API call.
+
+      **Note**: Datadog overrides the DTSTART attribute of a rule if it is passed.
```

Comment:
Maybe `Attributes specifying the duration in RRULE are not supported, e.g. DTSTART, DTEND, DURATION.`